clang/test/OpenMP/scan_ast_print.cpp

```diff
@@ -12,28 +12,6 @@
 
 void foo() {}
 
-template <class T>
```

Comment:
How was this test case working till now when templates were not supported in scan?

Reply:
The test was passing because the `for` directive at line:18 is used outside a parallel region, in which case the compiler wasn't crashing. Contrary to what would happen when used within a parallel region. 
Since the `scan` directive currently doesn't support templates, within or without a parallel region, I transferred this test to the scan_messages.cpp line:87 with expected errors.